src/examples/cpp/MyApp1.cpp

```diff
@@ -1,11 +1,11 @@
-#include <log4cxx/logger.h>
+#include <log4cxx/logmanager.h>
 #include <log4cxx/basicconfigurator.h>
 
-static auto logger = log4cxx::Logger::getLogger("MyApp");
```

Comment:
is there a particular reason that we should use `LogManager::getLogger` instead of `Logger::getLogger`?  IIRC they're effectively the same.